csrankings-r.csv

```diff
@@ -1013,6 +1013,10 @@ Roberto M. Negrini,Politecnico di Milano,http://www.deib.polimi.it/eng/people/de
 Roberto Manduchi,Univ. of California - Santa Cruz,https://users.soe.ucsc.edu/~manduchi,7YDSuZEAAAAJ
 Roberto Marcondes Cesar Jr.,USP,http://www.ime.usp.br/~cesar,NOSCHOLARPAGE
 Roberto Marcondes Cesar Junior,USP,http://www.ime.usp.br/~cesar,NOSCHOLARPAGE
+Roberto Martín-Martín,University of Texas at Austin,https://robertomartinmartin.com/,XOJE8OEAAAAJ
+Roberto Martin-Martin,University of Texas at Austin,https://robertomartinmartin.com/,XOJE8OEAAAAJ
+Roberto Martín Martín,University of Texas at Austin,https://robertomartinmartin.com/,XOJE8OEAAAAJ
+Roberto Martin Martin,University of Texas at Austin,https://robertomartinmartin.com/,XOJE8OEAAAAJ

```

Comment:
There should only be two entries, corresponding to the ones used by DBLP: Roberto Martin Martin, and Roberto Martín-Martín.

Reply:
Changed